engine/src/gfx/sprite.h

```diff
@@ -47,7 +51,7 @@ class VSSprite {
     bool isAnimation;
 
 //For private use only
-    VSSprite() : surface(0) {
+    VSSprite() : surface(nullptr), isAnimation(false), rotation(0.0F), maxs(0.0F), maxt(0.0F), heighto2(0.0F), widtho2(0.0F), xcenter(0.0F), ycenter(0.0F) {
```

Comment:
OK. Thanks. I think I might just leave it like this for now, in this particular case, since the code is already written. But I'll keep that in mind for the future.